omod/src/main/webapp/resources/htmlforms/Walkin-Client-Data-Capture-tool.xml

```diff
@@ -0,0 +1,243 @@
+                                        <label>National ID Number</label>
+                                        <span id="nin-number">
+                                            <patient field="identifier" identifierTypeId="f0c16a6d-dc5f-4118-a803-616d0075d282" required="false" />
+                                        </span>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+
+                        <div class="card">
+                            <div class="card-header">
+                                Vitals
+                            </div>
+                            <div class="card-body">                        
+                                <div class="row">
+                                    <div class="col-md-3">
+                                        <label>Weight</label>
+                                        <obs conceptId="5089" />
+                                        <label>Blood Pressure</label>
+                                        <div class="row ml-1">
+                                            <obs conceptId="5085" id="bp_systolic" />
+                                            <p class="left">
+                                                /
+                                            </p>
+                                            <obs conceptId="5086" id="bp_diastolic" />
+                                        </div>
+                                        <label>Temperature (c)</label>
+                                        <obs conceptId="CIEL:5088" id="temperature" />
+                                        
+                                    </div>
+                                </div>
+                                
+                            </div>
+                        </div>
+
+                        <div class="card">
+                            <div class="card-header">
+                                Clinical Assessment and Refill
+                            </div>
+                            <div class="card-body">
+                                <div class="row">
+                                    <div class="col-md-4">
+                                        <div class="row">
+                                            <div class="col-md-12">
+                                                <label>TB Pills</label>
+                                                <obs conceptId="166464" />
+                                                <label>INH Status</label>
+                                                <obs id="tpt-status" conceptId="165288" answerConceptIds="1090,90072,165307,90040,160035" answerLabels="H0-Never Initiated,H1-On INH,H2-Defaulted,H3-Taken off due to side effects,H4-Completed" />
```

Comment:
@ODORA0 Add a space between the dashes for the codes and their descriptions

Reply:
@ssmusoke Done